src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java

```diff
@@ -130,4 +145,11 @@ private List<HumanReadablePopulationModel> buildPopulations(Group group) {
   List<HumanReadableExpressionModel> buildDefinitions(Measure measure) {
     return List.of(new HumanReadableExpressionModel());
   }
+
+  private String escapeStr(String val) {
+    if (val != null && !val.isEmpty()) {
```

Comment:
maybe use `StringUtils.isNotBlank` or `StringUtils.isBlank` from apache commons

Reply:
I realized we don't even need this for referenceText, so Removed it. htmlEscape does handle nulls and empty strings. in our case, referenceText is required fields and it will never be null